examples/my-first-outreach-app/localhost.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Should we really commit the certificates ?

Btw there is a really nice tool at [mkcert.dev](https://mkcert.dev/) that allows to easily create certs for any local project

Reply:
sure, I'll add another step, which will create the certificates instead

Reply:
I've removed the keys from the repo and updated readme file!
Also I decided to use `openssl` isntead of `mkcert.dev` since that's what I've using in this redocly doc, so I thought it'd be better to stay consistent